pysap/info.py

```diff
@@ -90,5 +90,5 @@
 }
 PLUGINS = [
     "pysap-astro==0.0.0",
-    "pysap-mri==0.1.1"
+    "pysap-mri>=0.2.2"
```

Comment:
Actually this should be:

```python
"pysap-mri==0.2.2"
```

As we only want to use fixed releases of the plugins.

Reply:
Hmm, but wouldnt that just mean a PySAP release for every plugin release?
Although I do understand how this could otherwise be a potential issue. Changing this.

Reply:
You are right, but we can imagine having fixed release cycles where all plugins can be updated to the latest versions. Otherwise any plugin can break the installation, which could be a mess.